e107_languages/English/admin/lan_theme.php

```diff
@@ -106,5 +112,11 @@
 define("TPVLAN_CONV_14", "Enable this stylesheet as a selectable option in the Theme Manager.");
 define("TPVLAN_CONV_15", "Give this stylesheet a name");
 
+//marketplace
+define("EPL_ADLAN_94","Connecting...");
+define("EPL_ADLAN_233","Downloading..."); 
+define("EPL_ADLAN_234","Download Complete!");
+define("EPL_ADLAN_235","Automated download not possible. Please");
+define("EPL_ADLAN_236","Download Manually");
```

Comment:
EPL_ADLAN_236 should be "Please Download Manually" and "Please." removed from the previous LAN. Thank you! :+1: